core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1737,7 +1864,8 @@ public Channel asChannel() {
       return new SubchannelChannel(
           subchannel, balancerRpcExecutorHolder.getExecutor(),
           transportFactory.getScheduledExecutorService(),
-          callTracerFactory.create());
+          callTracerFactory.create(),
+          configSelector);
```

Comment:
I think for the moment this also needs to avoid inheriting the config selector. Yes, it is on a connection to a legit backend, but there's actually no LB policy in place. That means it would break xDS reference counting. It doesn't look like this previously used service config, so I think we should leave it config-less for now.

Reply:
Fixed.